frontend/public/components/app-contents.tsx

```diff
@@ -55,24 +55,22 @@ type DefaultPageProps = {
 
 // The default page component lets us connect to flags without connecting the entire App.
 const DefaultPage_: React.FC<DefaultPageProps> = ({ flags, activePerspective }) => {
-  const openshiftFlag = flags[FLAGS.OPENSHIFT];
-
-  if (flagPending(openshiftFlag)) {
+  if (Object.keys(flags).some(key => flagPending(flags[key]))) {
```

Comment:
It's a shame we have to block on all flags now. We might revisit this as a bug fix.